apps/backend/src/users/users.dto.ts

```diff
@@ -0,0 +1,39 @@
+import { Status, Role, Team } from './types';
+import {
+  IsEmail,
+  IsOptional,
+  IsEnum,
+  IsArray,
+  ArrayMinSize,
+  ArrayUnique,
+} from 'class-validator';
+
+export class UpdateUserDTO {
+  @IsOptional()
+  @IsEnum(Status)
+  status?: Status;
+
+  @IsOptional()
+  @IsEmail()
+  email?: string;
+
+  @IsOptional()
+  profilePicture?: string;
+
+  @IsOptional()
+  linkedIn?: string;
```

Comment:
:pick: for `linkedIn` and `github` properties, can we also make sure that they're links (i.e. `@IsUrl()`)?